tests/FakeItEasy.Analyzer.Tests.Shared/Helpers/CodeFixVerifier.Helper.cs

```diff
@@ -0,0 +1,84 @@
+﻿namespace FakeItEasy.Analyzer.Tests.Helpers
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading;
+    using Microsoft.CodeAnalysis;
+    using Microsoft.CodeAnalysis.CodeActions;
+    using Microsoft.CodeAnalysis.Formatting;
+    using Microsoft.CodeAnalysis.Simplification;
+
+    /// <summary>
+    /// Diagnostic Producer class with extra methods dealing with applying codefixes
+    /// All methods are static
+    /// </summary>
+    public abstract partial class CodeFixVerifier

```

Comment:
This class comes from the default template for analyzer projects. I just made the same adjustments as in `DiagnosticVerifier` (i.e. fix code analysis warnings and use FluentAssertions)